android/src/main/java/com/oney/WebRTCModule/CameraCaptureController.java

```diff
@@ -75,7 +86,30 @@ protected VideoCapturer createVideoCapturer() {
         String deviceId = ReactBridgeUtil.getMapStrValue(this.constraints, "deviceId");
         String facingMode = ReactBridgeUtil.getMapStrValue(this.constraints, "facingMode");
 
-        return createVideoCapturer(deviceId, facingMode);
+        Pair<String, VideoCapturer> result = createVideoCapturer(deviceId, facingMode);
+        if(result == null) {
+            return null;
+        }
+
+        String cameraName = result.first;
+        VideoCapturer videoCapturer = result.second;
+
+        // Find actual capture format.
+        Size actualSize = null;
+        if (videoCapturer instanceof Camera1Capturer) {
```

Comment:
This will make it extra harder to inject a custom capturer for those who want UVC support, for example.

I was thinking about having some RNCameraEnumerator (and an accompanying interface) for example, which would abstract the camera 1 and 2 implementations. This behavior could perhaps live there.